### PR TITLE
Rebuild for incomplete / missing specs

### PR DESCRIPTION
**Pull request by mrackwitz**
See CocoaPods/CocoaPods#3360.

### CODE REVIEW

File: install_custom_module_name/after/Pods/Pods.xcodeproj.yaml [hunk 1]
```diff
@@ -109,16 +109,15 @@ Targets:
           DYLIB_INSTALL_NAME_BASE: '@rpath'
           MODULEMAP_FILE: Target Support Files/Pods/Pods.modulemap
           CURRENT_PROJECT_VERSION: '1'
+          OTHER_LIBTOOLFLAGS: ''
```
Comment (neonichu) [inline install_custom_module_name/after/Pods/Pods.xcodeproj.yaml]: Wondering why this is here - won't it override the flags from the XCConfigs?

File: install_custom_module_name/after/Pods/Pods.xcodeproj.yaml [hunk 2]
```diff
@@ -109,16 +109,15 @@ Targets:
...
           MTL_ENABLE_DEBUG_INFO: 'YES'
+          OTHER_LDFLAGS: ''
```
Comment (neonichu) [inline install_custom_module_name/after/Pods/Pods.xcodeproj.yaml]: Wondering why this is here - won't it override the flags from the XCConfigs?

File: install_custom_module_name/after/Pods/Pods.xcodeproj.yaml [hunk 3]
```diff
@@ -135,16 +134,15 @@ Targets:
           DYLIB_INSTALL_NAME_BASE: '@rpath'
           MODULEMAP_FILE: Target Support Files/Pods/Pods.modulemap
           CURRENT_PROJECT_VERSION: '1'
+          OTHER_LIBTOOLFLAGS: ''
```
Comment (neonichu) [inline install_custom_module_name/after/Pods/Pods.xcodeproj.yaml]: Wondering why this is here - won't it override the flags from the XCConfigs?

File: install_custom_module_name/after/Pods/Pods.xcodeproj.yaml [hunk 4]
```diff
@@ -135,16 +134,15 @@ Targets:
...
           MTL_ENABLE_DEBUG_INFO: 'NO'
+          OTHER_LDFLAGS: ''
```
Comment (neonichu) [inline install_custom_module_name/after/Pods/Pods.xcodeproj.yaml]: Wondering why this is here - won't it override the flags from the XCConfigs?

File: install_custom_module_name/after/Pods/Pods.xcodeproj.yaml (hunk 1 above)
Comment (mrackwitz) [inline install_custom_module_name/after/Pods/Pods.xcodeproj.yaml]: It does override the flags form the XCConfings, [intentionally](https://github.com/CocoaPods/CocoaPods/blob/master/lib/cocoapods/installer/target_installer/aggregate_target_installer.rb#L35-47).

Comment (neonichu) [inline install_custom_module_name/after/Pods/Pods.xcodeproj.yaml]: Ah, awesome => :white_check_mark:

**Comment by neonichu** [comment]
LGTM :+1:

**Pull request merged**